tests/lib/external/snapd-testing-tools/tools/tests.pkgs.apt.sh

```diff
@@ -79,3 +79,8 @@ cmd_remove() {
     # shellcheck disable=SC2086
     apt-get remove --yes $REMOVE_FLAGS "$@"
 }
+
+cmd_download() {
+    # shellcheck disable=SC2068
+    apt download -q $@ >/dev/null
```

Comment:
@sergiocazzolato you *always* quote "$@"

```suggestion
    apt download -q "$@" >/dev/null
```

Reply:
updated